<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ajouterClient</class>
 <widget class="QMainWindow" name="ajouterClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>693</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>838</width>
    <height>693</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>838</width>
    <height>693</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Ajouter Client</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>logo-domo.png</normaloff>logo-domo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="nom">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Nom</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="prenom">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>40</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Prenom</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="mail">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>40</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>MÃ©l</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>761</width>
      <height>141</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="placeholderText">
     <string>Adresse</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="femme">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>250</y>
      <width>31</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>F</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="homme">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>250</y>
      <width>41</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>H</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>250</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Date Naissance</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateNaissance">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>250</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="numeroECU">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>250</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Numero ECU</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="poids">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>350</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Poids (pas obligatoire)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="taille">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>350</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Taille en cm (pas obligatoire)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="numeroSecours">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>350</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Numero Secours (pas obligatoire)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="numeroMedecin">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>390</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Numero medecin (pas obligatoire)</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>310</y>
      <width>118</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="numeroMedecin_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>390</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Numero secu (pas obligatoire)</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>450</y>
      <width>761</width>
      <height>151</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="placeholderText">
     <string>pathologies (pas obligatoire)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ajouter">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>630</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ajouter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="annuler">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>630</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Annuler</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>140</y>
      <width>741</width>
      <height>601</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="pixmap">
     <pixmap>../build-wifiEditor-Desktop_Qt_5_9_6_MinGW_32bit-Debug/logo-domo3.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>630</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>nom</zorder>
   <zorder>prenom</zorder>
   <zorder>mail</zorder>
   <zorder>textEdit</zorder>
   <zorder>femme</zorder>
   <zorder>homme</zorder>
   <zorder>label</zorder>
   <zorder>dateNaissance</zorder>
   <zorder>numeroECU</zorder>
   <zorder>poids</zorder>
   <zorder>taille</zorder>
   <zorder>numeroSecours</zorder>
   <zorder>numeroMedecin</zorder>
   <zorder>line</zorder>
   <zorder>numeroMedecin_2</zorder>
   <zorder>textEdit_2</zorder>
   <zorder>ajouter</zorder>
   <zorder>annuler</zorder>
   <zorder>progressBar</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
